pinot-core/src/main/java/org/apache/pinot/core/io/reader/impl/v1/FixedBitSingleValueReader.java

```diff
@@ -20,15 +20,15 @@
 
 import org.apache.pinot.core.io.reader.BaseSingleColumnSingleValueReader;
 import org.apache.pinot.core.io.reader.ReaderContext;
-import org.apache.pinot.core.io.util.FixedBitIntReaderWriter;
+import org.apache.pinot.core.io.util.FixedBitIntReaderWriterV2;
 import org.apache.pinot.core.segment.memory.PinotDataBuffer;
 
 
 public final class FixedBitSingleValueReader extends BaseSingleColumnSingleValueReader {
-  private final FixedBitIntReaderWriter _reader;
+  private final FixedBitIntReaderWriterV2 _reader;
```

Comment:
Directly use `PinotBitSet`, and move the logic of bulk read in `FixedBitIntReaderWriterV2` to this class.